kafka-spring-boot-autoconfigure/src/main/java/org/axonframework/extensions/kafka/autoconfig/KafkaAutoConfiguration.java

```diff
@@ -148,4 +152,41 @@ public Fetcher kafkaFetcher(ConsumerFactory<String, byte[]> kafkaConsumerFactory
     public KafkaMessageSource kafkaMessageSource(Fetcher kafkaFetcher) {
         return new KafkaMessageSource(kafkaFetcher);
     }
+
+    @ConditionalOnBean({ EventProcessingConfigurer.class })
+    @Autowired
+    public void configureKafkaEventProcessor(
+        final EventProcessingConfigurer eventProcessingConfigurer,
+        final KafkaProperties kafkaProperties) {
+
+        final EventProcessorMode mode = kafkaProperties.getEventProcessorMode();
+        switch (mode) {
+            case SUBSCRIBING:
+                eventProcessingConfigurer.registerSubscribingEventProcessor(KafkaSendingEventHandler.GROUP);
```

Comment:
Let us elaborate this issue. In general I see two possibilities:

* Sending all messages to multiple topics instead of one is a streaming operation and should be ideally not executed by the sender. Given the fact that as a sender, sending every message twice doubles the chance of getting an error and drives the developer in a complex error handling partial results (sent to one topic, but not to other, how to recover from this). In general, I vote for a streaming solution of this (having a splitter component sitting on the topic duplicating all messages to another topic) instead of solving it in the source.

* Despite the first possibility, we could invest in the technical solution allowing for registering multiple processors sending messages to multiple topics. For doing so, I would suggest to create a `KafkaPublisherConfiguration` interface and let the user provide implementation of that. This interface would contain the `ProducerPropeties`, the `Producer` and the configuration for the Axon Processing (processing group, mode). The extension would grab for all implementation of this interface available in the Spring context and register the corresponding processors. I estimate the effort of implementing this to half a day, but I'm afraid that we are doing things more complicated then they should be for an edge case. It is up to you to create an issue out of this - but please let us handle this case independent from this PR.

Reply:
> but please let us handle this case independent from this PR.

I completely agree with you here! 
I mostly wanted to get your stance on my hypothetical requirement and what that might entail. 
So, I am on the same track as you are, just wanted to hear your thoughts on the matter; thanks for sharing, as always. :+1: